services/UserService.js

```diff
@@ -1,26 +1,9 @@
 let userList = require("../db/UsersList");
```

Comment:
It seems you don't need `UsersList.js` file anymore because now you have a real database